modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/ModalAddObjectField.tsx

```diff
@@ -236,25 +239,25 @@ export function ModalAddObjectField({
 									values={values}
 								/>
 							)}
-						</ClayModal.Body>
+						</ClayForm>
+					</ClayModal.Body>
 
-						<ClayModal.Footer
-							last={
-								<ClayButton.Group spaced>
-									<ClayButton
-										displayType="secondary"
-										onClick={() => onClose()}
-									>
-										{Liferay.Language.get('cancel')}
-									</ClayButton>
+					<ClayModal.Footer
+						last={
+							<ClayButton.Group spaced>
+								<ClayButton
+									displayType="secondary"
+									onClick={() => onClose()}
+								>
+									{Liferay.Language.get('cancel')}
+								</ClayButton>
 
-									<ClayButton type="submit">
-										{Liferay.Language.get('save')}
-									</ClayButton>
-								</ClayButton.Group>
-							}
-						/>
-					</ClayForm>
+								<ClayButton type="submit">
```

Comment:
Hi @Artartur , currently I don't think handleSubmit will be called when you click the submit button. For this to work you need to add `form={modalAddObjectField}` to the ClayButton. This is what ties the submit button to the form.

Reply:
I see, ok, done! @ethib137